style.css

```diff
@@ -1,9 +1,437 @@
+  min-height: 900px;
+  justify-content: center;
+}
+
+.main_photo {
+  height: 436px;
+  width: 609px;
+}
+
+.column {
+  display: flex;
+  width: 609px;
+  align-items: flex-end;
+}
+
+.catch_phrase {
+  display: flex;
+  flex-direction: column;
+  width: 605px;
+  padding-bottom: 65px;
+
+}
+
+.invite_phrase {
+  margin-block-start: 0em;
+  margin-block-end: 0em;
+}
+
+.catch_phrase .invite_phrase {
+  text-align: left;
+  font-family: 'Roboto', sans-serif;
+  font-size: 96px;
+  font-weight: 400;
+
+}
+
+.photo {
+  padding-bottom: 65px;
+
+}
+
+.button_container {
+  display: flex;
+  justify-content: flex-start;
+  width: 609px;
+
+}
+
+.btn {
+  border-radius: 32px;
+  padding: 9px 30px;
+  font-size: 16px;
+  background-color: #D9DCD7;
+
+}
+
+/* TODO: order styles by sections/content */
+.second_page_container {
+  padding-bottom: 120px;
+}
+
+.section_title {
+  margin: 88px 0;
+  font-size: 36px;
+  font-weight: 400;
+  font-size: 36px;
+  text-align: center;
+  line-height: 119.5%;
+}
+
+.all_pros_container {
+  display: flex;
+  justify-content: center;
+}
+
+.high_way {
+  margin: 0px 8px;
+}
+
+
+.paragraph_header {
+  font-weight: bold;
+
+
+}
+
+.pros_photo {
+  width: 400px;
+  height: 324px;
+}
+
+.paragraph_description {
+  width: 400px;
+  display: flex;
+  justify-content: center;
+
+}
+
+.header_container {
+  display: flex;
+  justify-content: center;
+  width: 400px;
+}
+
+.container_for_paragraph {
+  height: 200px;
+  display: flex;
+  justify-content: center;
+}
+
+.text {
+  opacity: 0.8;
+  text-align: center;
+}
+
+
+
+
+.third_page_container {
+  display: flex;
+}
+
+.left_side {
+  display: flex;
+  height: 811px;
+  width: 712px;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  background-color: #ECE0E0;
+}
+
+.right_side {
+  height: 811px;
+  width: 712px;
+}
+
+.mountains_pic {
+  height: 811px;
+  width: 712px;
+}
+
+.some_text {
+  width: 400px;
+  height: 220px;
+  text-align: left;
+  font-family: 'Roboto', sans-serif;
+}
+
+.longer_paragraph {
+  opacity: 0.7;
+  font-size: 17px;
+}
+
+.third_page_header {
+  padding-left: 8px;
+  font-size: 25.5px;
+  font-weight: normal;
+}
+
+.btn_third_sector {
+  width: 400px;
+}
+
+.book_now {
+  border-radius: 32px;
+  width: 130px;
+  height: 35px;
+  background-color: #ECE0E0;
+
+}
+
+.book {
+  font-size: 16px;
+}
+
+.fourth_page_container {
+  display: flex;
+}
+
+.girl_pic {
+  width: 504px;
+  height: 604px;
+}
+
+.left_side_fourth_page {
+  display: flex;
+  height: 704px;
+  align-items: flex-end;
+}
+
+.right_side_fourth_page {
+  width: 936px;
+  height: 704px;
+  display: flex;
+  flex-direction: column;
+
+
+
+}
+
+.fan_number {
+  width: 936px;
+  height: 302px;
+  display: flex;
+  align-items: center;
+
+
+}
+
+.cabin_lovers {
+  font-size: 48px;
+  font-weight: normal;
+  font-family: 'Roboto', sans-serif;
+  margin-block-start: 0em;
+  margin-block-end: 0em;
+  margin-inline-start: 100px;
+  margin-inline-end: 0px;
```

Comment:
just set `margin: 0;`

Reply:
and don't use those margin declarations. Just `margin`

Reply:
fixed roger :)

Reply:
when you fix it in one place check other where it might appear :)

Reply:
fixed :) had to miss one :)